lib/table.ex

```diff
@@ -19,12 +19,20 @@ defmodule SnowflexTable do
   """
   @spec generate(integer) :: {:ok, integer} | :error
   def generate(node_id) do
-    if node_id > 0 or node_id < SnowflexId.node_limit() do
-      seq_num = :ets.update_counter(@table, node_id, @increment_op, {node_id, 0})
+    cond do
```

Comment:
these doesn't have the same semantics.
cond here won't help because there are only two conditions (true or false)
but I could see a bug tks (: